locales/it-IT.json

```diff
@@ -1,1927 +1,1927 @@
         "0": [
           "Prendi il traghetto."
-        ], 
+        ],
         "1": [
           "Prendi il Millersburg traghetto."
-        ], 
+        ],
         "2": [
           "Prendi il Bridgeport - Port Jefferson traghetto."
         ]
       }
-    }, 
+    },
     "enter_ferry_verbal": {
       "phrases": {
-        "0": "Prendi il traghetto.", 
-        "1": "Prendi il <STREET_NAMES>.", 
+        "0": "Prendi il traghetto.",
+        "1": "Prendi il <STREET_NAMES>.",
         "2": "Prendi il <STREET_NAMES> <FERRY_LABEL>."
-      }, 
+      },
       "empty_street_name_labels": [
-        "il marciapiedi", 
-        "la pista ciclabile", 
+        "il marciapiedi",
+        "la pista ciclabile",
         "la pista per mountain bike"
-      ], 
-      "ferry_label": "traghetto", 
+      ],
+      "ferry_label": "traghetto",
       "example_phrases": {
         "0": [
           "Prendi il traghetto."
-        ], 
+        ],
         "1": [
           "Prendi il traghetto Millersburg."
-        ], 
+        ],
         "2": [
           "Prendi il traghetto Bridgeport - Port Jefferson."
         ]
       }
-    }, 
+    },
     "enter_roundabout": {
       "phrases": {
-        "0": "Entra nella rotonda.", 
+        "0": "Entra nella rotonda.",
         "1": "Entra nella rotonda e prendi la <ORDINAL_VALUE> uscita."
-      }, 
+      },
       "ordinal_values": [
-        "1°", 
-        "2°", 
-        "3°", 
-        "4°", 
-        "5°", 
-        "6°", 
-        "7°", 
-        "8°", 
-        "9°", 
+        "1°",
+        "2°",
+        "3°",
+        "4°",
+        "5°",
+        "6°",
+        "7°",
+        "8°",
+        "9°",
         "10°"
-      ], 
+      ],
       "example_phrases": {
         "0": [
           "Entra nella rotonda."
-        ], 
+        ],
         "1": [
-          "Entra nella rotonda e prendi la 1° uscita.", 
-          "Entra nella rotonda e prendi la 2° uscita.", 
-          "Entra nella rotonda e prendi la 3° uscita.", 
-          "Entra nella rotonda e prendi la 4° uscita.", 
-          "Entra nella rotonda e prendi la 5° uscita.", 
-          "Entra nella rotonda e prendi la 6° uscita.", 
-          "Entra nella rotonda e prendi la 7° uscita.", 
-          "Entra nella rotonda e prendi la 8° uscita.", 
-          "Entra nella rotonda e prendi la 9° uscita.", 
+          "Entra nella rotonda e prendi la 1° uscita.",
+          "Entra nella rotonda e prendi la 2° uscita.",
+          "Entra nella rotonda e prendi la 3° uscita.",
+          "Entra nella rotonda e prendi la 4° uscita.",
+          "Entra nella rotonda e prendi la 5° uscita.",
+          "Entra nella rotonda e prendi la 6° uscita.",
+          "Entra nella rotonda e prendi la 7° uscita.",
+          "Entra nella rotonda e prendi la 8° uscita.",
+          "Entra nella rotonda e prendi la 9° uscita.",
           "Entra nella rotonda e prendi la 10° uscita."
         ]
       }
-    }, 
+    },
     "enter_roundabout_verbal": {
```

Comment:
@paolomi notice the `verbal` on the end of this one? this is the one meant for TTS engines

Reply:
@kevinkreiser 
ah ok, now I've understood! :-)  It's ok! many thanks! :-)

Reply:
@paolomi, we use verbal strings for TTS, so everything should be fine then. 

I would like to use a chance and thank everyone behind Valhalla - it's a great project and we are using it as a part of full offline navigation solution on Sailfish OS (other components include MapboxGL and libpostal). Thank you very much and good luck with the new endeavor at Mapbox!

Reply:
thank you so much @rinigus we're very happy to hear that everything worked out with your project!